dali/python/nvidia/dali/external_source.py

```diff
@@ -296,13 +330,27 @@ class ExternalSource():
     of separate Tensors, there will be an additional copy made internally, consuming both memory
     and bandwidth.
 
+    Cannot be explicitly set to ``False`` when used with ``parallel = True``
```

Comment:
```suggestion
    Cannot be explicitly set to ``True`` when used with ``parallel = True``
```?
As I understand no_copy doesn't work in such case, now I read it as it is enforced.

Reply:
It is enforced.

Reply:
And sort of hidden from the user, cause in paralle=True mode external source takes care of keeping batch in memory long enough to pass through the cpu stage.

Reply:
So maybe something like
```
Automatically set to ``True`` when ``parallel = True``
```

Reply:
done